src/main/webapp/go.js

```diff
@@ -105,7 +108,12 @@ function addScriptToHead() {
  * to reflect the current state of the hunt.
  */
 function getHunt() {
-  fetch(DATA_URL).then((response) => response.json()).then((mssg) => {
+  // ID of the scavenger hunt that the user is on,
+  // in the form of: hunt_id=ScavengerHunt([ID number here])
+  // Using index 1 because index 0 is the '?' character.
+  huntID = window.location.search.substr(1);
```

Comment:
Optional:
```suggestion
  huntID = new URLSearchParams(window.location.search).get('hunt_id');
```
is an option for a more structured way to get the value. Though then you'd have to explicitly put the `hunt_id=` part in to the `queryStr`.